src/proto/Ontology.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package TW.Ontology.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// Input data necessary to create a signed transaction.
+message SigningInput {
+
+    string contract = 1;
+
+    string method = 2;
+
+    string from_private_key = 3;
```

Comment:
Sorry, I overlooked this part, I think we should use bytes for `from_private_key` and `payer_private_key`, so we don't need to encode and parse the hex strings on both c++ and swift side

Reply:
okay, I will update this file as soon as possible.

Reply:
Here is the new `Ontology.proto`file: https://github.com/TrustWallet/wallet-core/pull/201/commits/5d1ce9f698e836f68a4a526b618692a55a9f513b